client/src/components/Home/ActivityStream.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Composer from 'components/Chat'
```

Comment:
Any reason this is no longer named `ChatInput`?

Reply:
I didn't understand the name ChatInput at first glance, so I tried to find a more explicit name. We can be even more explicit with MessageComposer. I also think that the file of this component should be renamed, because Chat seems too generic for that. A future PR… I can still keep ChatInput if you prefer ?